src/PostgrestBuilder.ts

```diff
@@ -106,9 +106,23 @@ export default abstract class PostgrestBuilder<Result>
 
         try {
           error = JSON.parse(body)
+
+          // Workaround for https://github.com/supabase/postgrest-js/issues/295
+          if (Array.isArray(error) && res.status === 404) {
```

Comment:
@soedirgo 
I was working on adding this update to the Flutter library and have some questions. 

What is this `Array.isArray(error)` check for? Is checking `res.status === 404` not enough in this case?

Reply:
I think that patch is no longer necessary even on `postgrest-js` since https://github.com/PostgREST/postgrest/issues/2343 is fixed and already deployed to all projects.

Reply:
@steve-chavez 
That is amazing! Was this deployed very recently? I had a user report this 2 days ago [here](https://github.com/supabase/supabase-flutter/issues/315).